model/storage/utils.py

```diff
@@ -2,10 +2,11 @@
 from model.data import ModelId
 
 
-# TODO make this configurable.
-def get_local_miner_dir(hotkey: str) -> str:
-    return os.path.join("local-models", hotkey)
+def get_local_miner_dir(base_dir: str, hotkey: str) -> str:
```

Comment:
General question: Where do we ensure the relevant directories are created?

Reply:
They get created the first time a file is written. So if we don't have local-models and we write to local-models/hotkey0/namespace/model/files then it creates all of those as needed.